.circleci/config.yml

```diff
@@ -18,7 +18,7 @@ parameters:
 
   node-version:
     type: string
-    default: 20.12-browsers
+    default: 20.17-browsers
```

Comment:
this was a bit odd - the `Dockerfile` is running against `20.17` but we're testing against `20.12` 😱.  Guess that's what happens when you let renovate do the upgrades 🤷